### DISCUSSION TITLE
Delete results from the DB and Visual view

### DISCUSSION THREAD

**Discussion by Fijxu** [cea-sec/ivre#1205] [Q&A] [answered]
Is there a way to remove results that have already been added to the database and to the visual view?
Because I accidentally added a lot of useless results and I would like to see if there is a way to remove or delete them

**Comment by p-l-** [comment, marked as answer]
From the CLI, you can run `ivre view --delete` and then add a filter (e.g., `ivre view --delete 192.168.0.0/16` would remove everything in the 192.168.0.0/16 network).

**Reply by Fijxu** [reply]
Thanks ;)
Is it possible to delete the results from the database and not only from the graphical view?
Also, is it possible to specify a category instead of an ip range as you said in the example?

**Reply by p-l-** [reply]
Yes and yes.

If by "from the database", you mean from the "scan" purpose, just run the same command with `ivre scancli`: `ivre scancli --delete 192.168.0.0/16`.

You can use any CLI filter: `ivre view --delete --category TEST` (and `ivre scancli --delete --category TEST`)
